examples/chip-tool/commands/clusters/WriteAttributeCommand.h

```diff
@@ -91,7 +96,7 @@ class WriteAttribute : public ModelCommand, public chip::app::WriteClient::Callb
         ChipLogProgress(chipTool, "Sending WriteAttribute to cluster " ChipLogFormatMEI " on endpoint %" PRIu16,
                         ChipLogValueMEI(clusterId), endpointId);
         chip::app::AttributePathParams attributePathParams;
-        if (!device->GetSecureSession().Value()->IsGroupSession())
+        if (device->GetSecureSession().Value()->IsGroupSession())
```

Comment:
Why this change?  This does not seem right.

Reply:
Oups.